modules/gds_mongodb/manifests/init.pp

```diff
@@ -1,5 +1,22 @@
+# == Class: gds_mongodb
+#
 # class to configure a replica set
-class gds_mongodb($members, $replSet) {
+#
+# === Parameters
+# [*members*]
+#   Individual hosts in a replicaset. Defined as an array.
+#
+# [*replSet*]
+#   Name of the replicaset.
+#
+# [*service*]
+#   MongoDB-Server daemon name.
+#
+class gds_mongodb(
+  $members,
+  $replSet,
+  $service
```

Comment:
Should we be forcing the service name? Seems like an odd thing to have to specify.

Also missing a comma.